webui/react/src/types.ts

```diff
@@ -183,16 +208,38 @@ export enum CheckpointState {
 }
 
 export interface Checkpoint {
+  endTime? : string;
   id: number;
+  resources: Record<string, number>;
+  startTime: string;
+  state: CheckpointState;
   stepId: number;
   trialId: number;
-  state: CheckpointState;
-  startTime: string;
-  endTime? : string;
   uuid? : string;
   validationMetric? : number;
 }
 
+export interface CheckpointDetail extends Checkpoint {
+  batch: number;
+  experimentId: number;
+  trialId: number;
+}
+
+export interface TrialSummary {
+  bestAvailableCheckpoint?: Checkpoint;
+  endTime?: string;
+  experimentId: number;
+  hparams: Record<string, string>;
+  id: number;
+  numBatches: number;
+  numBatchTally: number;
+  numCompletedCheckpoints: number;
+  numSteps: number;
+  seed: number;
```

Comment:
isn't this a trialDetail field?

Reply:
update: never mind it looks like now we are sending full trial details in /experiments/1/summary endpoint and not a trimmed down trial response. :confused: 

should we rename the types?